analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -190,15 +192,35 @@ class SpdxDocumentFile(
             Package(
                 id = pkg.toIdentifier(),
                 declaredLicenses = sortedSetOf(pkg.licenseDeclared),
-                concludedLicense = pkg.licenseConcluded.toSpdx(),
+                concludedLicense = getConcludedLicense(pkg),
                 description = packageDescription,
                 homepageUrl = pkg.homepage.mapNotPresentToEmpty(),
                 binaryArtifact = RemoteArtifact.EMPTY, // TODO: Use "downloadLocation" or "externalRefs"?
-                sourceArtifact = RemoteArtifact.EMPTY, // TODO: Use "sourceInfo" or "externalRefs"?
+                sourceArtifact = buildSourceArtifact(pkg),
                 vcs = VersionControlSystem.forDirectory(packageDir)?.getInfo() ?: VcsInfo.EMPTY
             )
         }
 
         return listOf(ProjectAnalyzerResult(project, packages))
     }
+
+    /**
+     * Return the concluded license to be used in ORT's data model, which expects a not present value to be null
+     * instead of NONE or NOASSERTION.
+     */
+    private fun getConcludedLicense(pkg: SpdxPackage): SpdxExpression? =
+        pkg.licenseConcluded.takeIf { SpdxConstants.isPresent(it) }?.toSpdx()
+
+    /**
+     * Return a [RemoteArtifact] created with downloadLocation of the given package [SpdxPackage]
+     * if it's a file location or return en Empty RemoteArtifact.
+     * TODO: must be completed for other cases : use "sourceInfo" or "externalRefs"?
+     */
+    private fun buildSourceArtifact(pkg: SpdxPackage): RemoteArtifact {
+        return if (pkg.downloadLocation.startsWith("file:/")) {
```

Comment:
I think if one slash is included in the comparision, the another one should be included for the same reason. E.g. use `file://`.

Reply:
@fb33  this is not yet solved...

Reply:
Hello, I follow the same idea as in Downloader:267
```
        // Some (Linux) file URIs do not start with "file://" but look like "file:/opt/android-sdk-linux".
        val sourceArchive = if (pkg.sourceArtifact.url.startsWith("file:/")) {
            File(URI(pkg.sourceArtifact.url))
```
But also, if I use file:// (ex: `file://C:/workspace/temp/legacy-test/libs/deps1/mydll.dll`) it crashes with 
```
Exception in thread "main" java.lang.IllegalArgumentException: URI has an authority component
        at java.base/java.io.File.<init>(File.java:426)
        at org.ossreviewtoolkit.downloader.Downloader.downloadSourceArtifact(Downloader.kt:269)
```
But I have no error if I use `file:/C:/workspace/temp/legacy-test/libs/deps1/mydll.dll`

Reply:
Have you tried `file:///C:/` ?

Reply:
Yes, you're right. It works with ///
But the comment in Download :  "for some (linux) file URIs..." is still correct ?

Reply:
I have no clue what comment you mean.

Reply:
In downloader/src/main/kotlin/Downloader.kt line 267
```
 // Some (Linux) file URIs do not start with "file://" but look like "file:/opt/android-sdk-linux".
        val sourceArchive = if (pkg.sourceArtifact.url.startsWith("file:/")) {
            File(URI(pkg.sourceArtifact.url))
```

Reply:
I'm fine with merging this as-is to document the idea. File-based "downloads" require more work anyway to work completely.

@fviernau if you agree please dismiss your review / approve.